docs/psoc6/quickref.rst

```diff
@@ -411,8 +412,17 @@ MISO    P9_1           P6_1         P10_1
 SCK     P9_2           P6_2         P10_2
 =====  ===========  ============  ============
 
+.. 
+
+    TODO: This is only applicable to the CY8CPROTO-062-4343W. This does not belong here. 
```

Comment:
I agree this is specific to only proto board. But also, in mpy official docs other ports which support multiple boards still have I guess the most common board reference (pinouts etc.) in quickref. But we will have to move around our docs a bit. Maybe have common port docs and point out to getting started with different boards or something.

Reply:
Agree, other ports show only one board. We have to discuss this in the strategy, and find our compromise for maintainability - user friendliness.